mw-webapp/src/assets/icons/SunIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import {IconProps} from "src/component/icon/Icon";
+
+/**
+ * Sun icon
+ */
+export const SunIcon = (props: IconProps) => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 24 24"
+      fill="none"
+      className={props.className}
+    >
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M12 0.300003C12.2761 0.300003 12.5 0.523861 12.5 0.800003V4C12.5 4.27615 12.2761 4.5 12
+          4.5C11.7239 4.5 11.5 4.27615 11.5 4V0.800003C11.5 0.523861 11.7239 0.300003 12
+          0.300003ZM3.72685 3.72685C3.92211 3.53159 4.2387 3.53159 4.43396 3.72685L6.6967
+          5.9896C6.89196 6.18486 6.89196 6.50144 6.6967 6.6967C6.50144 6.89196 6.18485
+          6.89196 5.98959 6.6967L3.72685 4.43396C3.53159 4.2387 3.53159 3.92212 3.72685
+          3.72685ZM0.8 11.5C0.523858 11.5 0.3 11.7239 0.3 12C0.3 12.2761 0.523858 12.5
+          0.8 12.5H4C4.27614 12.5 4.5 12.2761 4.5 12C4.5 11.7239 4.27614 11.5 4
+          11.5H0.8ZM3.72685 20.2732C3.53159 20.0779 3.53159 19.7613 3.72685 19.566L5.98959
+          17.3033C6.18485 17.108 6.50144 17.108 6.6967 17.3033C6.89196 17.4986 6.89196 17.8151
+          6.6967 18.0104L4.43396 20.2732C4.2387 20.4684 3.92211 20.4684 3.72685 20.2732ZM20
+          11.5C19.7239 11.5 19.5 11.7239 19.5 12C19.5 12.2761 19.7239 12.5 20 12.5H23.2C23.4761
+          12.5 23.7 12.2761 23.7 12C23.7 11.7239 23.4761 11.5 23.2 11.5H20ZM17.3033 6.6967C17.108
+          6.50144 17.108 6.18486 17.3033 5.9896L19.566 3.72685C19.7613 3.53159 20.0779 3.53159
+          20.2732 3.72685C20.4684 3.92212 20.4684 4.2387 20.2732 4.43396L18.0104 6.6967C17.8151
+          6.89196 17.4986 6.89196 17.3033 6.6967ZM12.5 20C12.5 19.7239 12.2761 19.5 12 19.5C11.7239
+          19.5 11.5 19.7239 11.5 20V23.2C11.5 23.4761 11.7239 23.7 12 23.7C12.2761 23.7 12.5 23.4761
+          12.5 23.2V20ZM17.3033 17.3033C17.4986 17.108 17.8151 17.108 18.0104 17.3033L20.2732
+          19.566C20.4684 19.7613 20.4684 20.0779 20.2732 20.2732C20.0779 20.4684 19.7613 20.4684
+          19.566 20.2732L17.3033 18.0104C17.108 17.8151 17.108 17.4986 17.3033 17.3033ZM8.5 12C8.5
+          10.067 10.067 8.5 12 8.5C13.933 8.5 15.5 10.067 15.5 12C15.5 13.933 13.933 15.5 12
+          15.5C10.067 15.5 8.5 13.933 8.5 12ZM12 7.5C9.51472 7.5 7.5 9.51472 7.5 12C7.5 14.4853
+          9.51472 16.5 12 16.5C14.4853 16.5 16.5 14.4853 16.5 12C16.5 9.51472 14.4853 7.5 12 7.5Z"
+        strokeWidth="1"
```

Comment:
In moon icon strokeWidth === 2 and in eye icons too. Maybe better to update sunicon's strokewidth too?

Reply:
sunicon's strokeWidth="1"

![02](https://github.com/tritonJS826/masters-way/assets/112805668/8b0384da-d185-412f-941a-1fa7f3bf154a)

sunicon's strokeWidth="2

![03](https://github.com/tritonJS826/masters-way/assets/112805668/029a90db-42ce-4ca7-b1d2-d63881b1ea6f)
-------
visually the sunicon is more thicker with strokeWidth="2